scripts/prettier/index.js

```diff
@@ -37,7 +37,7 @@ const files = glob
   .filter(f => !onlyChanged || changedFiles.has(f));
 
 if (!files.length) {
-  return;
+  process.exit(0);
```

Comment:
prettier didn't like the top level return. Used `process.exit(0)` instead to replicate the behavior (see line 82 is also using `process.exit` already).